### ISSUE TITLE
Currently Available Compatible Sound Cards

### ISSUE THREAD

**Issue by MrShigura** [pideck/pideck#72]
Just wanted to check if there are any pi sound cards other than the audio injector (16ms latency is pushing it, at best, for scratching) that have been confirmed to work with pideck, as the ones mentioned that have 
 been confirmed seem to be discontinued and/or no longer sold now. I've read the required specifications for what kind of card should work, but would be helpful to just have one or two specific cards (preferably board base/hat/internal) that are known to generally be compatible.

**Comment by obbardc** [comment]
I think we can get much better latency with a real-time kernel. The audio
injector dual is the best option right now, I think. We were talking about
a custom version with phono preamp & onboard passthru switch.

We’re eeally busy on other projects right now, but we will get back to this
soon!

**Comment by MrShigura** [comment]
Thanks for the response, and for the work you've done on this project so far. Just to make sure, are you referring to:

https://www.amazon.com/dp/B01HCC0210

If so, roughly what latency is achievable with that card and a Pi 3 B+?

**Comment by obbardc** [comment]
That’s the one, probably better than 8ms with an RT kernel..
It’s on our list to remake the image for 3B+, with a rt kernel, but we only
tried a 3B.

**Comment by Captainkorg** [comment]
sounds realy nice.
custom version with phono preamp & onboard passthru switch.
I also think about it but my ideas go to buy an extra phono preamp like this
https://www.ebay.de/itm/Velleman-Bausatz-K2573-RIAA-Stereo-Vorverstaerker-Hifi-Audio-Endstufe-/191748379225

But this will be need an extra powersupply and so on.
Currently i use the audio-injector (the Soundcard works fine for me) its the best option according to the pidisplay and smartipitouch 
![img_4005](https://user-images.githubusercontent.com/10151200/38133673-7cb76cf4-3410-11e8-8d24-167ea5e8df33.JPG)

Will your sollution for the preamp smarter? :) so i would save my money and keep waiting. How you will promote updates?
Best 
Gunnar

**Comment by obbardc** [comment]
Looks awesome, great job. Basically that is our exact setup...

I'd say get the velleman board and give it a go. There are probably
cheaper ones from China if you can wait a while ;-). It doesn't need
to be Hifi quality.



PS: I've just made a google group, sign up to it here & enable email
notifications.....

https://groups.google.com/d/forum/pideck

**Comment by danielhjames** [comment]
On the USB front, this looks interesting but seems to have limited availability: http://www.mclellandmusic.com/en_productdetail.php?id=106

The Behringer UFO202 is still available, if you don't mind making a small hardware mod to cut a couple of traces on the circuit board.

In theory, we will get lower latency than USB using the audioinjector.

**Comment by Captainkorg** [comment]
The Phono AMP looks good but its out of my range of Investment for this Application (i want to keep it under 200€ Display+Frame was not inexpensiv).
I think with an USB Booth and an smal preamp i get more succes in the direction what i want.
the aim is one Power supply and one device.
:) even for an easy to use dj mix between digital an vinyl.

**Comment by MrShigura** [comment]
@danielhjames I actually have a Behringer 404hd sitting here currently unused, but I'm guessing that's outside of what pi can support. 

There's another project which shall remain nameless I'm sure you're aware of that's using an audio injector of one kind or another, but I'm not sure what kind of latency said unnamed project is getting

**Comment by MrShigura** [comment]
In retrospect, I realize I made my question more detailed than both it needed to be, and than I am really knowledgeable enough to process the answer to -_-

Given that, let me rephrase...

What would your recommendation be for a sound card for the pi 3 B+ (I guess additionally, will the 3 B+ work with the current build...) I can go purchase right now that should give me close to the best latency I'm going to get at the moment, and that can be expected to come down in latency when the updates you mentioned do happen. Basically I just need a "buy this" answer so I can stop procrastinating... =P

ps: I'll try to get that data from the 62 asap


@obbardc @danielhjames

**Comment by obbardc** [comment]
Get the AudioInjector dual :-)

The current image does not work with the 3B+, but it's on our to do list....


Chris

**Comment by danielhjames** [comment]
In the meantime, your Behringer 404HD should work if you use phono pre-amps or (maybe, just maybe) software pre-amplification. See https://linuxmusicians.com/viewtopic.php?t=17231

**Comment by MrShigura** [comment]
@danielhjames @obbardc Appreciate the replies, as all I have atm is a pi 3 B+ appears I'll have to wait until there's a compatible version, but sounds like when there is should be fairly easy to get set up.

**Comment by obbardc** [comment]
It won't be long, promise :-)....

In the mean time, it _may_ work with the current image, can you give it a
go?

**Comment by MrShigura** [comment]
@obbardc Sure, I'll get my hands on the audio injector first as I'm aiming for a... um... small form factor solution let's say for now... and see what happens

**Comment by MrShigura** [comment]
@obbardc @danielhjames Tried the current image with the Pi 3 B+, no go. Possible I did something wrong, but the image just never tries to load at all, blank screen, nothing.

**Comment by obbardc** [comment]
@MrShigura we have a new image for testing. please try: http://64studio.net/~chris/pideck/

PS: there are lots of things wrong with the image ATM. feel free to comment with your findings...

Cheers!

**Comment by danielhjames** [comment]
I have created a soundcard matrix at https://github.com/pideck/pideck/blob/master/soundcards.md
